lib/executor/helpers.go

```diff
@@ -33,6 +33,32 @@ func getStagesUnscaledMaxTarget(unscaledStartValue int64, stages []Stage) int64
 	return max
 }
 
+func validateNumberOfVuShifts(startVus int64, stages []Stage) []error {
+	const MaxRampingVUShift = 100000000
```

Comment:
Is there a specific reason for having this constant defined in the function body? Otherwise, I would suggest moving this constant out of the function scope and probably keeping it unexported (name starting with a lowercase letter). What do you think? Opinions on this @codebien ?